kgforge/specializations/models/rdf/rdf_model_service_from_store.py

```diff
@@ -115,24 +117,27 @@ def recursive_resolve(self, context: Union[Dict, List, str]) -> Dict:
                 context.remove(self.store_metadata_iri)
             if hasattr(self.default_store.service, "store_local_context") and\
                     self.default_store.service.store_local_context in context:
+
                 context.remove(self.default_store.service.store_local_context)
             for x in context:
                 document.update(self.recursive_resolve(x))
         elif isinstance(context, str):
+
             try:
-                local_only = False if self.default_store == self.context_store else True
+                local_only = not self.default_store == self.context_store
                 doc = self.default_store.service.resolve_context(context, local_only=local_only)
             except ValueError:
                 try:
                     doc = self.context_store.service.resolve_context(context, local_only=False)
                 except ValueError as e:
                     raise e
+
             document.update(self.recursive_resolve(doc))
         elif isinstance(context, dict):
             document.update(context)
         return document
 
-    def _load_shape(self, resource_id):
+    def _load_shape(self, resource_id: URIRef):
```

Comment:
Why resource_id is turned into URIRef here ? store.retrieve expects an str

Reply:
My mistake, just changed it